pay-api/src/pay_api/services/eft_short_names.py

```diff
@@ -96,18 +97,33 @@ def create_shortname_link(cls, short_name_id: int, auth_account_id: str, **kwarg
         current_app.logger.debug('>create_shortname_link')
         return cls.find_link_by_id(eft_short_name_link.id)
 
+    @classmethod
+    def delete_shortname_link(cls, short_name_link_id: int):
+        """Delete EFT short name auth account link."""
+        current_app.logger.debug('<delete_shortname_link')
+        short_name_link: EFTShortnameLinksModel = EFTShortnameLinksModel.find_by_id(short_name_link_id)
+
+        if short_name_link.status_code != EFTShortnameStatus.PENDING.value:
+            raise BusinessException(Error.EFT_SHORT_NAME_LINK_INVALID_STATUS)
+
+        short_name_link.delete()
```

Comment:
versioning table works with this? 

do we capture this hard delete somehow?

Reply:
What happens after a short name link delete? Is there somewhere else we need to record this?

Reply:
As discussed another ticket  for unlink / refund